pkg/asset/machines/aws/machines.go

```diff
@@ -54,6 +54,9 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 					"sigs.k8s.io/cluster-api-machine-role": role,
 					"sigs.k8s.io/cluster-api-machine-type": role,
 				},
+				Annotations: map[string]string{
+					"openshift.io/drain-node": "True",
```

Comment:
hmm, `True` vs `true` ?

Reply:
@abhinavdahiya agree on the awkwardness...

see:
https://github.com/openshift/cluster-api-provider-aws/pull/135/files#diff-acdcd15b24a848e93a682408b101755fR257

@ingvagabund we may want to make case-insensitive.

Reply:
PTAL: https://github.com/openshift/cluster-api-provider-aws/pull/144